frontend/src/App.tsx

```diff
@@ -93,6 +99,11 @@ const App = (): React.ReactElement => {
                   path={Routes.AUTH_ACTIONS}
                   component={AuthActions}
                 />
+                <Route
+                  exact={false}
```

Comment:
What's the different between the two routes you added? Shouldn't we always use exact matching?

Reply:
deleted the inexact route